.gitlab-ci.yml

```
@@ -27,10 +28,16 @@ variables:
   DEVEL_BUILD_DIR: /tmp/conda_artefacts/devel
   DEVEL_COMPARE_BRANCH: master
   #
+  # master commits
+  #
+  MASTER_CHANNEL_REM: s3://dnap/npg/conda/prod/generic
+  MASTER_CHANNEL_DIR: /tmp/prod/generic
+  MASTER_BUILD_DIR: /tmp/conda_artefacts/prod
+  MASTER_COMPARE_BRANCH: old_master
+  #
   # TEST
   #
   TEST_ENV_PREFIX: $CONDA_DIR/envs/test_env
-  PROD_WSI_CONDA_CHANNEL: https://dnap.cog.sanger.ac.uk/npg/conda/prod/generic
 
 include:
   - .gitlab_include/mirror.yml
```

```suggestion
  - .gitlab_include/mirror.yml
  - .gitlab_include/master_commit.yml
```
Does this need adding in this section?